### DISCUSSION TITLE
Documentation - Community don't work.

### DISCUSSION THREAD

**Discussion by kriss145** [nuxt/nuxt#27087] [Kind words]
Hello everyone,

I don't know where to send so I create this discussion. 

One of the pages in the Documentation doesn't work: https://nuxt.com/docs/community/getting-help 

![screen-error](https://github.com/nuxt/nuxt/assets/3781356/13853305-aeb4-4bf0-9dd8-1d9b046314d5)

I wanted to check a roadmap.

**Comment by meirroth** [comment]
@kriss145 On what page did you find this link?

**Comment by xjccc** [comment]
I think docs maybe re-publish at that time. Please refresh your browser, it would be solve.

**Comment by TheAlexLichter** [comment]
Let's track in https://github.com/nuxt/nuxt.com/issues/1569
